Public/Src/Cache/DistributedCache.Host/Service/Internal/DistributedContentStoreFactory.cs

```diff
@@ -354,11 +354,12 @@ private AzureBlobStorageCredentials[] GetStorageCredentials(Dictionary<string, S
             return credentials.ToArray();
         }
 
-        private static AzureBlobStorageCredentials CreateAzureBlobCredentialsFromSasToken(UpdatingSasToken updatingSasToken)
+        private AzureBlobStorageCredentials CreateAzureBlobCredentialsFromSasToken(string secretName, UpdatingSasToken updatingSasToken)
         {
             var storageCredentials = new StorageCredentials(sasToken: updatingSasToken.Token.Token);
-            updatingSasToken.TokenUpdated += (token, sasToken) =>
+            updatingSasToken.TokenUpdated += (updatingToken, sasToken) =>

```

Comment:
I think the first argument here is a sender, so I think it make sense to rename the argument.

Reply:
Sine you are not using the variable you can use `_` i.e.

```suggestion
updatingSasToken.TokenUpdated += (_, sasToken) =>
```

---
In reply to: [310297477](https://github.com/microsoft/BuildXL/pull/698#discussion_r310297477) [](ancestors = 310297477)